step.sh

```diff
@@ -344,7 +345,11 @@ else
 fi
 
 # SonarQube
-sonarScannerOptions="-X -Dsonar.host.url=${sonar_host_url} -Dsonar.login=${SONAR_HOST_LOGIN} -Dsonar.projectKey=${project_key} -Dsonar.language=swift -Dsonar.exclusions=${exclusions} -Dsonar.organization=${sonar_host_organization} -Dsonar.projectVersion=${projet_version}"
+if [ "$enableDebug" = "on"]; then
+    sonarScannerOptions="-X "
+fi
+
+sonarScannerOptions="-Dsonar.host.url=${sonar_host_url} -Dsonar.login=${SONAR_HOST_LOGIN} -Dsonar.projectKey=${project_key} -Dsonar.language=swift -Dsonar.exclusions=${exclusions} -Dsonar.organization=${sonar_host_organization} -Dsonar.projectVersion=${projet_version}"
```

Comment:
@SarahJmaiel The "-X " will be overwritten…

Reply:
problem fixed